python/lsst/ip/isr/linearize.py

```diff
@@ -283,15 +284,15 @@ def fromTable(cls, tableList):
         metadata = coeffTable.meta
         inDict = dict()
         inDict['metadata'] = metadata
-        inDict['hasLinearity'] = metadata['HAS_LINEARITY']
+        inDict['hasLinearity'] = metadata.get('HAS_LINEARITY', False)
         inDict['amplifiers'] = dict()
 
         for record in coeffTable:
             ampName = record['AMPLIFIER_NAME']
 
-            fitParams = record['FIT_PARAMS'] if 'FIT_PARAMS' in record else None
-            fitParamsErr = record['FIT_PARAMS_ERR'] if 'FIT_PARAMS_ERR' in record else None
-            fitChiSq = record['RED_CHI_SQ'] if 'RED_CHI_SQ' in record else None
+            fitParams = record['FIT_PARAMS'] if 'FIT_PARAMS' in record.columns else np.array([0.0])
+            fitParamsErr = record['FIT_PARAMS_ERR'] if 'FIT_PARAMS_ERR' in record.columns else np.array([0.0])
+            fitChiSq = record['RED_CHI_SQ'] if 'RED_CHI_SQ' in record.columns else np.nan
```

Comment:
I'm sure there's a good reason for changing all these sentinel values here and above, so that's probably fine, but in a vacuum I certainly prefer `None` to zero and nan (but like I say, I'm sure there's a good reason, I've just not dug into it here).

Reply:
`None` does not serialize well with astropy tables (where it becomes `astropy.io.fits.card.Undefined`).  I also wanted to keep datatypes consistent for the downstream code.